packages/slice-machine/lib/mock/Slice.ts

```diff
@@ -74,7 +74,10 @@ export default function MockSlice(
     }),
     type: SlicesTypes.SharedSlice,
   };
-  return sliceMockConfig.map((sc) =>
-    generateSliceMock(sliceModel, sc)(renderSliceMock)
-  );
+  // console.dir({legacyMockConfig, model, sliceMockConfig, sliceModel}, {depth: null})
```

Comment:
are these comments left for future devs?

Reply:
nope, I just forgot to remove them :)

Reply:
..looks like i did remove them ?